bin/plugin/commands/performance.js

```diff
@@ -122,12 +122,14 @@ async function getPerformanceResultsForBranch(
 	environmentDirectory,
 	branch
 ) {
+	await git.discardLocalChanges( environmentDirectory );
```

Comment:
This ensures that if there's a package-lock.diff lingering it's discarded.

Reply:
> This ensures that if there's a package-lock.diff lingering it's discarded.

Could make sense as an inline comment.

```suggestion
	// Restore clean working directory (e.g. if `package-lock.json` has local
	// changes after install).
	await git.discardLocalChanges( environmentDirectory );
```